tests/test_add_basic.py

```
@@ -0,0 +1,59 @@
+        print(f"Result type: {type(result)}")
+        print("✓ PublicKey.add method executed successfully")
+        print(f"✓ Result is PublicKey instance: {isinstance(result, PublicKey)}")
+        print(f"✓ Result has same prefix: {result.prefix == test_pub.prefix}")
+
+        # Test that child method works
+        print("\nTesting child method...")
+        child_key = test_pub.child(b"test_offset")
+        print(f"Child key: {child_key}")
+        print("✓ PublicKey.child method executed successfully")
+
+        return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)